packaging/bin/engine-backup.sh.in

```diff
@@ -1138,6 +1257,12 @@ dorestore() {
 		generatePgPass
 		verifyConnection "${CINDERLIB_DB_USER}" "${CINDERLIB_DB_HOST}" "${CINDERLIB_DB_PORT}" "${CINDERLIB_DB_DATABASE}"
 	fi
+    if [ -n "${CHANGE_KEYCLOAK_DB_CREDENTIALS}" ]; then
```

Comment:
Please fix the indentation - this file uses TABs. Sadly github does not show this...

Reply:
fixed.